src/dotnet/UI.Blazor/Components/ContextMenu/BlazorContextMenu/Components/SubMenu.razor

```diff
@@ -0,0 +1,33 @@
+@using RenderFragment = Microsoft.AspNetCore.Components.RenderFragment
+@inherits ContextMenuBase
+@namespace BlazorContextMenu
+
+<div @attributes="@Attributes" class="@($"{BaseClass} {ClassCalc} {DisplayClassCalc}")" id="@Id" style="@($"left:{X}px;top:{Y}px;z-index:{ZIndex};")" data-autohide="@AutoHide.ToString().ToLower()">
+    @if (IsShowing)
+    {
+        <ul class="@ListClassCalc">
+            <CascadingValue Value="(MenuTreeComponent)this" Name="ParentComponent">
+                <CascadingValue Value="@(Template != null ? Template : CascadingTemplate)" Name="CascadingTemplate">
+                    <CascadingValue Value="@(Animation.HasValue ? Animation.Value : CascadingAnimation)" Name="CascadingAnimation">
+                        @ChildContent
+                    </CascadingValue>
+                </CascadingValue>
+            </CascadingValue>
+        </ul>
+    }
+</div>
+
+
+@code{
+    [Parameter]
+    public RenderFragment ChildContent { get; set; } = null!;
+    protected override string BaseClass => "blazor-context-submenu blazor-context-menu__wrapper";
+    protected override void OnInitialized()
+    {
+        if (string.IsNullOrEmpty(Id))
+        {
```

Comment:
Brackets should be on the same line inside methods + you actually don't need brackets here.